setup.cfg

```diff
@@ -0,0 +1,54 @@
+[metadata]
+name = pyfscacher
```

Comment:
may be let's rename the beast right away into `fscacher` (?   I named it with `py` originally to signify that we need Python solution, but all the `py` prefixes are not really needed, and typically it is better to have it uniform between project and package name, so let's rename this repository to `fscacher` as well.
Unless you have some counter arguments?  or may be even a better name overall?

Reply:
Package renamed.